<?xml version="1.0" ?><root>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Footjob'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Handjob'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_ShortTerm_Sex_AdditionalSexScore'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_ShortTer' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_ShortTer' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelTrack_StripClub_Charm'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_StripClub_Charm' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_StripClub_Charm' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Impregnated_SimB'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelTrack_Personality_Archetype_Impression'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_Personality_Arche' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_Personality_Arche' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_NoProtection'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Oraljob'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Anal'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Impregnated_SimA'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Vaginal'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:11:50</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelTrack_Relationship_WhiteRomance'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_Relationship_Whit' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_Relationship_Whit' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:21:55</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>posture_graph.py:5164</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>[rez] Posture graph threw an exception while being queried by autonomy: (AssertionError: Source spec not in the posture graph: stand@object_paintEaselGEN_01:0x0fe4042667a30000, carry, surface:slot_target@object_paintEaselGEN_01:0x0fe4042667a30000 for interaction:  running canvas_PaintPainting_Staging_Large:1675 on object_Canvas_large:0x075014639de80432  (guaranteed))&#13;&#10;Traceback (most recent call last):&#13;&#10;Caught and logged:&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 2988, in _get_transitions_for_sim&#13;&#10;  File "postures\posture_graph.py", line 5164, in postures.posture_graph.PostureGraphService.get_segmented_paths&#13;&#10;AssertionError: Source spec not in the posture graph: stand@object_paintEaselGEN_01:0x0fe4042667a30000, carry, surface:slot_target@object_paintEaselGEN_01:0x0fe4042667a30000 for interaction:  running canvas_PaintPainting_Staging_Large:1675 on object_Canvas_large:0x075014639de80432  (guaranteed)rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:25:57</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>persistence.py:109</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>Exception in Sim Timeline: Exception running Element (AttributeError: 'NoneType' object has no attribute 'get_sim_id')&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 231, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 335, in _run&#13;&#10;  File ".\WickedWhims_v178c\wickedwhims\sex\integral\routing\interactions.py", line 191, in _init_sex_interaction&#13;&#10;  File ".\WickedWhims_v178c\wickedwhims\sex\integral\routing\interactions.py", line 67, in _try_init_route_sex&#13;&#10;  File ".\WickedWhims_v178c\wickedwhims\sex\integral\sex_handlers\pre_sex\pre_sex_instance.py", line 211, in start_sex_interaction&#13;&#10;  File ".\WickedWhims_v178c\wickedwhims\sex\integral\persistence.py", line 57, in apply_sim_sex_variables&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\wrappers\sim\super_sim.py", line 72, in _safe_sim_info_wrapper&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\wrappers\sim\buffs.py", line 61, in add_buff&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\__init__.py", line 470, in exported_func&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\buff_component.py", line 312, in add_buff_from_op&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\buff_component.py", line 361, in add_buff&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\event_buffs.py", line 25, in on_add&#13;&#10;  File ".\WickedWhims_v178c\wickedwhims\sex\integral\persistence.py", line 109, in on_addition&#13;&#10;  File ".\WickedWhims_v178c\wickedwhims\sex\features\sex_mood_context\body_attraction.py", line 68, in apply_sex_mood_body_attraction_buffsAttributeError: 'NoneType' object has no attribute 'get_sim_id'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>504009b67435f08567e11340</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:26:58</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>pre_sex_instance.py:217</categoryid><desyncid>504009b67435f08567e11340</desyncid><systemconfig/><screenshot/><desyncdata>Exception in Sim Timeline: Exception running Element (AttributeError: 'NoneType' object has no attribute 'get_temp_value')&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 231, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 335, in _run&#13;&#10;  File ".\WickedWhims_v178c\wickedwhims\sex\integral\routing\interactions.py", line 191, in _init_sex_interaction&#13;&#10;  File ".\WickedWhims_v178c\wickedwhims\sex\integral\routing\interactions.py", line 67, in _try_init_route_sex&#13;&#10;  File ".\WickedWhims_v178c\wickedwhims\sex\integral\sex_handlers\pre_sex\pre_sex_instance.py", line 217, in start_sex_interaction&#13;&#10;  File ".\WickedWhims_v178c\wickedwhims\sex\integral\sex_handlers\active_sex\active_sex_instance.py", line 516, in play_if_everyone_readyAttributeError: 'NoneType' object has no attribute 'get_temp_value'rtim=0&#13;&#10;Client Session Time: 36.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
</root>